docs/public-networks/how-to/bonsai-limit-trie-logs.md

```diff
@@ -0,0 +1,187 @@
+---
+title: Limit Trie Logs for Bonsai
+sidebar_position: 12
+description: Enable this early access feature to reduce the size of your database
+tags:
+  - public networks
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Limit Trie Logs for Bonsai
+## Step by Step Guide
+
+:::caution
+
+We strongly recommend reading the rest of this documentation before running these commands because your node’s configuration may require changes to these commands.
+
+:::
+
+*Targeting 24.1.2 release, but currently you must be on the Besu `main` branch for this to work*
+
+1. Update Besu config to add --Xbonsai-limit-trie-logs-enabled but don’t restart yet
+1. Stop Besu
+1. (optional) Run: 
+`sudo /usr/local/bin/besu/bin/besu --data-storage-format=BONSAI --data-path=/var/lib/besu --sync-mode=X_SNAP storage x-trie-log prune`
+1. Start Besu (remembering to run sudo systemctl daemon-reload if you use a systemd service file as per CoinCashew and Somer)
+1. Look out for `Limit trie logs enabled: retention: 512; prune window: 30000` in your Besu config printout during startup
+1. Enjoy more GBs
+
+## What?
+We have a new experimental feature available: `--Xbonsai-limit-trie-logs-enabled` which aims to keep Besu’s database as small as it can be. After a brief grace period, we intend to make this enabled by default for stakers. This is only relevant if you're using `data-storage-format=BONSAI`.
```

Comment:
For tech docs, we typically don't write with a marketing tone and usually opt for language that is a but more succinct and to the point. I think a rewrite for this would be something like:

```suggestion
The feature `--Xbonsai-limit-trie-logs-enabled` minimizes the database size of Besu and requires the use of `data-storage-format=BONSAI`. When enabled, this feature can reduce database growth by more than 3 GB each week.
```